pkg/kubelet/apis/cri/runtime/v1alpha2/api.proto

```diff
@@ -1,3 +1,19 @@
+/*
+Copyright 2018 The Kubernetes Authors.
```

Comment:
This file was added in 2018: https://github.com/kubernetes/kubernetes/pull/58973.